### PR TITLE
Admin edit update tour

### PR DESCRIPTION
**Pull request by conh-2027**
>>> Title: 
Admin edit update tour
>>> Redmine ticket: 
https://edu-redmine.sun-asterisk.vn/issues/17714
>>> Attachment: 
Screenshot from 2019-09-20 19-16-34
Screenshot from 2019-09-24 14-16-08

### CODE REVIEW

**Comment by conh-2027** [comment]
ready

**Comment by conh-2027** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -50,9 +50,9 @@
 }
 
 .login_btn {
-  font-size: 24px;
-  font-family: sans-serif;
-  color: black;
+    font-size: 24px;
```
Comment (hungct-1121) [inline app/assets/stylesheets/custom.scss]: indent 2 space đang đúng

File: app/controllers/admin/tours_controller.rb [hunk 2]
```diff
@@ -20,9 +22,26 @@ def create
     end
   end
 
+  def update
+    if @tour.update tour_params
+      flash[:success] = t ".update_success"
+      redirect_to admin_tours_path
+    else
+      render :edit
```
Comment (hungct-1121) [inline app/controllers/admin/tours_controller.rb]: thêm flash đi

**Comment by conh-2027** [comment]
ready

**Comment by conh-2027** [comment]
ready

**Pull request merged**